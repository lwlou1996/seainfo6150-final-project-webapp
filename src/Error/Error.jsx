import React from 'react'
import notFound from "../images/error.png";
import NaviBar from "../Bar/NaviBar/NaviBar";

import styles from './Error.module.css'

const Error = () => {
    return (
        <div className={styles.container}>
            <div>
                <NaviBar />
            </div>

            <div className={styles.imgContainer}>
                <img className={styles.errorImg} src={notFound} alt="Page Not Found"/>
            </div>

        </div>
    )
}

export default Error
