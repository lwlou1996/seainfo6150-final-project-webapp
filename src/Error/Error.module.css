
.container {
    display: flex;
    flex-flow: column;
    /*justify-content: center;*/
}

.imgContainer {
    text-align: center;
}

.errorImg {
    width: 70%;
}
