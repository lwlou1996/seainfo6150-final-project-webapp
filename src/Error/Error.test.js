import React from 'react'
import {render} from "@testing-library/react";
import Error from "./Error";

describe('error page test', () => {

    it('render successfully ', () => {
        const {container} = render(<Error />);
        expect(container).toMatchSnapshot();
    });

})