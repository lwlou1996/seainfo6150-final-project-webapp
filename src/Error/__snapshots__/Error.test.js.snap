// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`error page test render successfully  1`] = `
<div>
  <div
    class="container"
  >
    <div>
      <div
        class="navContainer"
      >
        <header
          class="container"
        >
          <nav>
            <a
              class="mainIcon"
              href="/"
            >
              PoSub
            </a>
          </nav>
          <nav>
            <ul
              class="list"
            >
              <li
                class="listItem"
              >
                <a
                  href="/contact"
                >
                  Contact
                </a>
              </li>
              <li
                class="listItem"
              >
                <a
                  href="/category"
                >
                  Category
                </a>
              </li>
            </ul>
          </nav>
        </header>
      </div>
    </div>
    <div
      class="imgContainer"
    >
      <img
        alt="Page Not Found"
        class="errorImg"
        src="error.png"
      />
    </div>
  </div>
</div>
`;
